### DISCUSSION TITLE
Number of Contributions don't show on hover first time the page load

### DISCUSSION THREAD

**Discussion by anshul-5harma** [community/community#29092] [Other Feature Feedback, Questions, & Ideas] [answered]
It seems that the first time you open https://github.com/yourUsername you cannot see number of contributions made on any given day when you hover over contributions grid. See below for reference.
<img width="737" alt="Screen Shot 2022-08-10 at 8 41 21 PM" src="https://user-images.githubusercontent.com/27900079/184046356-0b75a695-93dd-4785-ab4e-4d40f55ce0fc.png">

When the page is reloaded, the functionality returns to normal i.e. on hover you are able to see number of contributions on any given day. See below for reference
<img width="740" alt="Screen Shot 2022-08-10 at 8 42 42 PM" src="https://user-images.githubusercontent.com/27900079/184046437-8dade06a-5d9c-4f34-9e10-a179f0faac41.png">

**Comment by radar-blip** [comment]
I've noticed this also, the team is probably already aware of it but you can file a bug report if you like, here:
https://support.github.com/contact/bug-report

**Comment by Dexalt142** [comment, marked as answer]
I've noticed this bug several days ago, but it's kinda different. If you open https://github.com/username directly, the functionality is normal. However, if you navigate the page, let's say from a repository page to your profile page, the contribution number is not showing up.

**Comment by timagixe** [comment]
I have faced the same issue.
